### DISCUSSION TITLE
Curator not deleting by disk space on True Cluster

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#8106] [Unsupported Versions] [answered]
Hi!
I have an installation with True cluster enabled, curator is running only on the manager (which should be expected [#2806](https://github.com/Security-Onion-Solutions/securityonion/issues/2806))
My problem is that curator is not deleting logs by disk space on the searchnodes.

is it possible that the fact is due to the pattern `'^(logstash-.*|so-.*)$'` in the delete.yml?
Would changing it to `'(logstash-.*|so-.*)$'` or `'^*:(logstash-.*|so-.*)$'` in `/opt/so/saltstack/local/salt/curator/files/action/delete.yml` solve the problem?

Then I should set a log_size_limit for the manager that considers the storage space of the entire cluster, correct?

**Comment by dougburks** [comment, marked as answer]
Please take a look at the Elastic Clustering section of our documentation and what it says about index deletion:
https://docs.securityonion.net/en/2.3/elasticsearch.html#elastic-clustering
